frontend/src/styles/colors.ts

```diff
@@ -0,0 +1,31 @@
+const colors = {
```

Comment:
All of of these colors are in `frontend/src/theme/foundations/colors.ts`. Could we delete this to avoid duplication? How are you using it?